docs/api/doc-block-subtitle.md

```diff
@@ -24,10 +24,16 @@ import { Subtitle } from '@storybook/blocks';
 
 `Subtitle` is configured with the following props:
 
+### `of`
```

Comment:
Thanks for updating the API reference! ❤️

Could you please keep these props in alphabetical order?

Reply:
Sorted :)